### DISCUSSION TITLE
How to return first api to modify with approve workflow

### DISCUSSION THREAD

**Discussion by CrawlFu** [elsa-workflows/elsa-core#2618] [Q&A]
hi, 
1. when submit form data send to email with manager approval ,manager don't approve the data ,need to modify the form data and resend email to approve , how to set the modify api .thanks.

this is my workflow 
![image](https://user-images.githubusercontent.com/24671924/146868846-9ef00080-d8db-4dff-b230-e25b0c3c36cc.png)

**Comment by sfmskywalker** [comment]
I'm not sure I understand your question completely, but if you already have the infrastructure in place to submit form data to the workflow, then I imagine you would use the same machinery to submit the same or another form again. Perhaps you can rephrase your question and be a little bit more specific?
